influxql/query/select_test.go

```diff
@@ -2881,6 +2884,10 @@ func TestSelect(t *testing.T) {
 		},
 	} {
 		t.Run(tt.name, func(t *testing.T) {
+			if tt.onlyArch != "" && runtime.GOARCH != tt.onlyArch {
```

Comment:
What's the scenario where `tt.onlyArch` is the empty string? Should this `&&` be `||` instead?

Reply:
Empty string is the default (i.e. expected to work on every GOARCH). `&&` is the right operator, it's basically saying "if a single GOARCH was specified AND we're not running on that arch"